tests/test_resource.py

```diff
@@ -104,123 +71,84 @@ def test_method(self):
         }
         get_user.assert_called_once_with({})
 
-    def test_not_found_returned_when_path_not_defined(self):
+    def test_not_found_returned_when_path_not_defined(self, sample_request) -> None:
+        event_wrong_uri = Event(
+            resource_path="/xxxs/asdasd/xxx",
+            method="GET",
+            headers={},
+            path_params={},
+            query_params={},
+            body=sample_request,
+        )
         response = Resource(event_wrong_uri)()
         assert isinstance(response, Response)
         assert response.status_code == HTTPStatus.NOT_FOUND
 
-    def test_request_id_added_when_frameworks_exception_raised(self):
+    def test_unsupported_method(self, sample_request) -> None:
+        event_wrong_method = Event(
+            resource_path="/",
+            method="" "PRINT",
+            headers={},
+            path_params={},
+            query_params={},
+            body=sample_request,
+        )
+        res = Resource(event_wrong_method)()
```

Comment:
Didn't you mean `resp` or even `response`? You are using `res` as a shortcut for `resource` in some places - it can be misleading 😉